shopno/src/styles/style.scss

```
@@ -0,0 +1,594 @@
+@import "./fonts.scss";
+@import "./popUp.scss";
+
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+html {
+    font-size: 10px;
+    @media (max-width: 1024px) {
+        font-size: 9.5px;
+    }
+    @media (max-width: 960px) {
+        font-size: 8px;
```

use rem

---

This is a reference size for html tag 
```suggestion
       html {
    font-size: 10px;
    @media (max-width: 1024px) {
        font-size: 9.5px;
    }
    @media (max-width: 960px) {
        font-size: 8px;
    }
}
```